src/04_printing.py

```diff
@@ -11,7 +11,10 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print('x is %d, y is %d, z is %s' % (x, float(y), z))
 
 # Use the 'format' string method to print the same thing
+print("x is {}, y is {}, z is {}".format(x, round(y, 2), z))
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+# Finally, print the same thing using an f-string
+print(f"x is %d, y is %.2f, z is %s" %(x, y, z))
```

Comment:
check out the new and improved way to format strings in python 3.6
https://realpython.com/python-f-strings/